Mykolb.txt

```diff
@@ -0,0 +1,9 @@
+Hi, my name is Mykol (pronounced Michael). 
+3 words to describe:
+    1. genuine
+    2. funny
+    3. patient 
+
+My favorite hobby is reading! Whoo-whoo!
+My favorite animal would be a tiger or any kind of cat.
```

Comment:
Nice - I have three cats and I probably will have a fourth soon.  
Great work.
This is done correctly.